draco/synthesis/mockcatalog.py

```diff
@@ -0,0 +1,1045 @@
+"""Tasks for making mock catalogs.
+
+See Usage section for usage.
+
+Pipeline tasks
+==============
+
+.. autosummary::
+    :toctree:
+
+    SelFuncEstimator
```

Comment:
Additionally, it might be better to just call this `SelectionFunctionEstimator` (same with the other tasks). It's not horrendously longer and it is clearer.